### PR TITLE
Fix ogbl-wikikg2 loader

### PR DESCRIPTION
**Pull request by brs96**
Thank you for your contribution to the Graph Data Science Client project.

Before submitting this PR, please read [Contributing to the Neo4j Ecosystem](https://github.com/neo4j/graph-data-science-client/blob/main/CONTRIBUTING.md). 

Make sure:
- [ ] You signed the [Neo4j CLA](https://neo4j.com/developer/cla/#sign-cla) (Contributor License Agreement) so that we are allowed to ship your code in our library
- [ ] Your contribution is covered by tests

### CODE REVIEW

**Comment by brs96** [comment]
[Cypher aggregation OGB graph fails with IllegalArgumentException: The node has to be either a NODE or an INTEGER, but got Double](https://trello.com/c/gDoYb8By/8122-cypher-aggregation-ogb-graph-fails-with-illegalargumentexception-the-node-has-to-be-either-a-node-or-an-integer-but-got-double)

File: graphdatascience/graph/ogb_loader.py [hunk 1]
```diff
@@ -48,6 +48,7 @@ class HomogeneousOGBNDataset(Protocol):
     # The representation is a node_count x 1 shaped matrix
     labels: npt.NDArray[np.int64]
     meta_info: "pd.Series[Any]"
+    name: str
```
Comment (adamnsch) [inline graphdatascience/graph/ogb_loader.py:51]: We don't need this attribute for the other Dataset protocols right?

File: graphdatascience/tests/integration/test_ogb_loader.py [hunk 2]
```diff
@@ -55,7 +53,24 @@ def test_graph_load_ogbn_mag(gds: GraphDataScience) -> None:
     G.drop()
 
 
-@pytest.mark.enterprise
+@pytest.mark.ogb
+@pytest.mark.compatible_with(min_inclusive=ServerVersion(2, 1, 0))
+def test_graph_load_ogbl_wikikg2(gds: GraphDataScience) -> None:
```
Comment (adamnsch) [inline graphdatascience/tests/integration/test_ogb_loader.py:58]: There are several more asserts I would like to have in this test (see `test_graph_load_ogbn_arxiv above`)

File: graphdatascience/tests/integration/test_ogb_loader.py [hunk 3]
```diff
@@ -55,7 +53,24 @@ def test_graph_load_ogbn_mag(gds: GraphDataScience) -> None:
...
+    # ogbl-wikikg2 is a homogenous dataset for knowledge graph completion
+
+    G = gds.graph.ogbl.load("ogbl-wikikg2")
+
+    assert G.name() == "ogbl-wikikg2"
+    assert G.node_count() == 2_500_604
+    # We don't store the negative sampled relationships in the graph.
+    assert G.relationship_count() == 17_137_181
+
+    # 535 train rels
```
Comment (adamnsch) [inline graphdatascience/tests/integration/test_ogb_loader.py:68]: ?

**Review by adamnsch — CHANGES_REQUESTED** [review]
Nice work :) Let me know if you want to discuss any of the comments

**Review by adamnsch — APPROVED** [review]

**Pull request merged**